plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -313,6 +316,14 @@ func (p *antreaOctantPlugin) actionHandler(request *service.ActionRequest) error
 			return nil
 		}
 		return nil
+	case runTraceAgain:
+		// Get name of new traceflow
+		mostRecentTraceflow.Name = mostRecentTraceflow.Spec.Source.Pod + "-" + mostRecentTraceflow.Spec.Destination.Pod 
+		mostRecentTraceflow.Name += "-" + time.Now().Format(TIME_FORMAT_YYYYMMDD_HHMMSS)
+
+		log.Printf("Ran traceflow again successfully, Traceflow Results:  %+v\n", mostRecentTraceflow)
```

Comment:
It hasn't run traceflow successfully, right? Maybe just remove this line if it doesn't provide more information than existing logs in `createTfCR`